src/components/application_manager/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) 2016, Ford Motor Company
```

Comment:
Alright, I've been told it doesn't matter either way, but I believe the way that I suggest would be cleaner